### ISSUE TITLE
Litecoin network data

### ISSUE THREAD

**Issue by Vasiliy-Bondarenko** [Bit-Wasp/bitcoin-php#610]
Hello. Litecoin uses M-addresses, but you [did not account for that yet](https://github.com/Bit-Wasp/bitcoin-php/blob/69378452293d3b48e15a018e4ca1df3ab9608ff1/src/Network/NetworkFactory.php#L85). Can you please check all network data?

https://github.com/litecoin-project/litecoin/pull/279

i think it should be 
```
    /**
     * @return NetworkInterface
     * @throws \Exception
     */
    public static function litecoin()
    {
        $network = self::create('30', '32', 'b0')
                       ->setHDPubByte('019da462')
                       ->setHDPrivByte('019d9cfe')
                       ->setNetMagicBytes('dbb6c0fb')
                       ->setSegwitBech32Prefix("ltc");

        return $network;
    }
```
but i'm not sure :)

after that changes i get M-addresses which are correct according to their converter https://litecoin-project.github.io/p2sh-convert/ - i've tested before- and after-addresses.

**Comment by afk11** [comment]
Oops, yes you're right! 

I've been unsure about the best way to do this. It's this stuff that makes me nervous about having altcoin networks in here, I sure don't keep as up to date with them as I do bitcoin.. and the people who PR them don't maintain it :P Could put both forms in there too so people can switch between them. 

I've started using named classes for networks now too in master, so eventually it'll be easier to identify a network instance.

**Comment by afk11** [comment]
It'll be a BC break, so unless we go for 0.0.35 (branching off of master, which is 7 only and has other changes)

**Comment by afk11** [comment]
https://github.com/litecoin-project/litecoin/blob/master/src/chainparams.cpp#L239
It looks like litecoin uses 111 = 0x6f in their chainparams.cpp (I normally refer to this if in doubt)

But indeed the converter you linked uses `0x3a`.. maybe @coblee could tie break?

**Comment by dtmax** [comment]
I'm talking about multisig addresses, sorry.
And seems there is 3A: https://github.com/litecoin-project/litecoin/blob/master/src/chainparams.cpp#L241
Also, I took that Q-address not from online-converter, but from server-side Litecoin 0.15 while doing multisig address:
`[api@api ~]$ ltc -conf=test addmultisigaddress 2 [\"025fd...01b22a067\",\"03ec....c450516be54\",\"03cf059e.....28e0f2\"]
Qg1HQQk6mtoHcnZDW2s2vNeUCVkNG5Gkev`

**Comment by afk11** [comment]
Of course, sorry my mistake. 

PR looks good, merged!

**Comment by afk11** [comment]
This should be available in v0.0.35.x, so closing this
